itests/kit/node_unmanaged.go

```diff
@@ -9,12 +17,15 @@ import (
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
+const sectorSize = abi.SectorSize(2 << 10) // 2KiB
```

Comment:
There's already a `sectorSize` in the test kit, I just couldn't figure out a way to feed that back, I wanted to ask the kit "what sector size am I using?" but I think 2k just gets hard-wired everywhere in itests now, which is a little unfortunate. Maybe we should make a `SectorSize()` method on the miner so we can: use the 2k default that the test kit uses, have a `SectorSize()` option, and also have a `SectorSize()` method on the miner so we can work with it.

But also this brought up another, adjacent, issue when I was looking at it, we have this line: https://github.com/filecoin-project/lotus/blob/6302607650f9191bcc471547c595d712875b2f0f/itests/kit/ensemble.go#L284-L286

But we also have `kit.TestSpt` and I'm adding `kit.TestSptNi`, so hard-wired seal proof types; I don't think they should be hard-wired but maybe we add new methods `SealProofType()` and `SealProofTypeNi()` to get them instead of these constants.

Reply:
this still needs to go, it's in the opts, we should only have it once

Reply:
Fixed this.